test/migrators/division/bad_paths_verbose.hrx

```diff
@@ -0,0 +1,12 @@
+<==> arguments
+--migrate-deps --verbose
+
+<==> input/entrypoint.scss
+@import "does_not_exist";
+@import "does_not_exist2";
+
+<==> log.txt
+WARNING - 2 dependencies could not be found.
+  does_not_exist @entrypoint.scss:1
```

Comment:
In verbose mode, I think the messages should include the full source span decoration.